### PR TITLE
3982 migration form confirmation

### LINKED ISSUE #3982: Forking: Migration Form, confirmation checkbox
If user migrates all their REP show small disclaimer

https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=2334%3A9652

Screen Shot 2019-10-02 at 10 11 39 AM

 - [x] If user migrates a portion of their REP show notice banner

image

### PR DESCRIPTION
**Pull request by bconfortin**
closes https://github.com/AugurProject/augur/issues/3982

Please check the conditions for disabling the button. I believe they are correct, but it's best to double check.
Tried not to mess with `disabled: this.props.market.marketType === SCALAR ? true : false`

Also, there's a change on the `top: 3px;` of the checkbox that I did last week. It's back at `top: 0;`.

Check not checked
Check not checked
Check checked
Check checked
Partial REP migration
Partial REP migration
Full REP migration
Full REP migration

### CODE REVIEW

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 1]
```diff
@@ -713,9 +706,40 @@ export class ReportingBondsView extends Component<
...
+              />
+              I have carefully read all the information and fully acknowledge the consequences of migrating my REP to an unsuccessful universe
+            </label>
+          </div>
+        )}
         <PrimaryButton
-          text="Confirm"
-          disabled={disabled}
+          text={migrateRep ? "Confirm and Migrate REP" : "Confirm"}
+          disabled={migrateRep ? !readAndAgreedCheckbox : disabled}
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx]: need to make sure there is no errors before enabling the button. 

issue here is, an error and button will still be active if they click the box.

**Review by bthaile — CHANGES_REQUESTED** [review]
small change needed, other than that looks good.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🆒

 This link will always point to this PR's latest changes: https://peek.run/o9ph7xw

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 2]
```diff
@@ -713,9 +706,44 @@ export class ReportingBondsView extends Component<
...
+            </label>
+          </div>
+        )}
         <PrimaryButton
-          text="Confirm"
-          disabled={disabled}
+          text={migrateRep ? "Confirm and Migrate REP" : "Confirm"}
+          disabled={
+            migrateRep
+              ? (disabled || (disabled || !readAndAgreedCheckbox))
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx]: ` ? (disabled || !readAndAgreedCheckbox))` this might work and be more readable

**Comment by bthaile** [comment]
once change request.
also make sure to test the reporting form for regular initial reporting and disputing
check for errors in web console

**Review by bthaile — APPROVED** [review]
changes look good

**Pull request merged**